### ISSUE TITLE
Add page localization

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-server#7127]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Current Limitation
It is currently not easily possible to localize pages for
- password reset
- email verification
- invalid link
- etc.

### Feature / Enhancement Description
Localize the Parse Server pages by handling an optional user locale in the query of the public API routes, such as
- `request_password_reset`
- `choose_password`
- `verify_email`
- `resend_verification_email`

With the introduction of [Parse Server API Mail Adapter](https://github.com/parse-community/parse-server/issues/7125), a user locale parameter can be attached to links being sent in the password reset and email verification emails, making it easy to attach a user locale in the query.

### Example Use Case

1. Send "Reset Password" email that includes link with locale in query, e.g.
   ```
   http://example.com/parse/apps/test/request_password_reset?token=exampleToken&username=exampleUsername&locale=de-AT'
   ```
2. In Parse Server `Public API Router` detect the locale and return a localized file.

### Alternatives / Workarounds
- Determining the user locale not from the query but from the username could expose the user locale (which is PII)

### 3rd Party References
- none

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/5.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0](https://github.com/parse-community/parse-server/releases/tag/5.0.0)
